prisma/schema.prisma

```diff
@@ -88,3 +90,14 @@ model Note {
   createdAt DateTime  @default(now())
   updatedAt DateTime? @updatedAt
 }
+
+model Comment {
+  id        Int       @id @default(autoincrement())
+  content   String    @db.VarChar(255)
```

Comment:
What was the reasoning behind setting a 255 character limit? Is this listed in the requirements?